### PR TITLE
Combine all

### PR DESCRIPTION
**Pull request by Farnazmdi**

### CODE REVIEW

**Comment by Farnazmdi** [comment]
Could you please take a look at the combination notebook in jenkins? @aarmey 
There are no more negative combination parameters, no more negative cell number, and made a test as you said in a new test file for combination.

**Comment by Farnazmdi** [comment]
If they seem ready, I can send out the notebooks to Drs. Heiser and Gross.

File: combination.ipynb [hunk 1]
```diff
@@ -0,0 +1,209 @@
+{
+ "cells": [
+  {
```
Comment (aarmey) [inline combination.ipynb]: Indicate time point of these plots. Some explanation of them at the top would be helpful, too.

File: src/Hill.jl [hunk 2]
```diff
@@ -138,7 +138,7 @@ end
 
 """ Calculate the # of cells in G1 for a set of parameters and T """
 function numcells(params, g0, T)
-    @assert(all(params .>= 0.0), "negative params $params")
+#     @assert(all(params .>= 0.0), "negative params $params")
```
Comment (aarmey) [inline src/Hill.jl]: Why comment this out?

File: combination.ipynb (hunk 1 above)
Comment (Farnazmdi) [reply combination.ipynb]: Sure.

File: src/Hill.jl (hunk 2 above)
Comment (Farnazmdi) [reply src/Hill.jl]: Oops, sorry forgot to uncomment, although I have another assert in the BlissCombination that does the same.

**Comment by Farnazmdi** [comment]
image

**Comment by Farnazmdi** [comment]
Should I send the notebooks to Heiser lab?

**Pull request merged**